config/routes.rb

```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
  root 'items#index'
- get  "/items/new" =>  "items#new"
- post "/items"     =>  "items#create"
+ get  "/items/new"                =>  "items#new"
+ post "/items"                    =>  "items#create"
+ get  "/items/:id/purchases/new"  =>  "purchases#new"
+ post "/items/:id/purchases"      =>  "purchases#create"
```

Comment:
This makes me think that a purchase belongs to an item, or that an item has many purchases.